### DISCUSSION TITLE
Unable to locate a class or view for component [filament-support::button]. - upgrading v2-to-v3

### DISCUSSION THREAD

**Discussion by erenilhan** [filamentphp/filament#7548] [Help] [answered]
Hi,

I tried to upgrade to v3 and it's done but I getting the error;

https://flareapp.io/share/17xD6Z97

`Unable to locate a class or view for component [filament-support::button].`

I tried to optimize:clear and cache:clear but no solution.

**Comment by danharrin** [comment, marked as answer]
Are you using `filament-support::button` in your codebase? It should be `filament::button`

**Comment by erenilhan** [comment]
yes, I published vendor files (v2) before. I changed it and it's working, thank you 🙏

**Reply by danharrin** [reply]
Please remove all published vendor files. We do not recommend using them.

**Reply by erenilhan** [reply]
I will, thank you again.
